ivy/functional/backends/jax/experimental/norms.py

```diff
@@ -1,21 +1,30 @@
 import jax.numpy as jnp
 from typing import Optional
 from ivy.functional.backends.jax import JaxArray
+from ivy.func_wrapper import with_supported_device_and_dtypes
+from . import backend_version
 
 
+@with_supported_device_and_dtypes(
+    {
+        "0.4.10 and below": {
+            "cpu": (
+                "float16",
+                "float32",
+                "float64",
+            )
+        }
+    },
+    backend_version,
+)
 def l1_normalize(
     x: JaxArray,
     /,
     *,
     axis: Optional[int] = None,
     out: Optional[JaxArray] = None,
 ) -> JaxArray:
-    if axis is None:
-        norm = jnp.sum(jnp.abs(x.flatten()))
-        denorm = norm * jnp.ones_like(x)
-    else:
-        norm = jnp.sum(jnp.abs(x), axis=axis, keepdims=True)
-        denorm = jnp.divide(norm, jnp.abs(x) + 1e-12)
+    denorm = jnp.linalg.norm(x, 1, axis, keepdims=True)
```

Comment:
You need to handle the case when `axis=None` as this function accepts None axes only when the array is less than 3D